extensions/raytasker/src/components/form.tsx

```diff
@@ -0,0 +1,107 @@
+import { Action, ActionPanel, Color, Form, Icon, showToast, Toast, useNavigation } from "@raycast/api";
+import { useState } from "react";
+import { FormProps, Task } from "../interfaces";
+import * as google from "./../oauth/google";
+
+export function EditTaskForm(props: FormProps) {
+  const { pop } = useNavigation();
+  const [isChanged, setIsChanged] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  return (
+    <Form
+      isLoading={isLoading}
+      navigationTitle={"Task"}
```

Comment:
Please don't change the [navigationTitle](https://developers.raycast.com/basics/prepare-an-extension-for-store#navigation-title) in the root command; it's getting the title from the command.